Sources/DependenciesMacrosPlugin/DependencyEndpointMacro.swift

```diff
@@ -256,3 +256,26 @@ extension String {
     return String(result)
   }
 }
+
+extension TupleTypeElementSyntax {
+  fileprivate var isAutoclosure: Bool {
+    self.type
+      .as(AttributedTypeSyntax.self)?
+      .attributes
+      .contains {
+        $0
+          .as(AttributeSyntax.self)?
+          .attributeName
+          .as(IdentifierTypeSyntax.self)?
+          .name
+          .tokenKind == .identifier("autoclosure")
```

Comment:
This code `.tokenKind` differs from #159. I'm thinking that `tokenKind` might be better than `text`, but I'll defer to your opinion.